bitnami/keycloak/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   category: DeveloperTools
 apiVersion: v2
-appVersion: 20.0.1
```

Comment:
The version was updated in a previous PR but that's wrong, we are not using 20.x right now, current version is 19.x. We will start using 20.x from now on